PocketKit/Sources/PocketKit/String+Localization.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension String {
+    func localized(withComment: String = "") -> String {
```

Comment:
@bassrock is it worth adding a breadcrumb for strings we've attempted to localize, but don't have strings for? or will that add clutter in dev while we wait for translations?

Reply:
@dskuza that will clutter dev while we wait for them. 

Let's ticket this as followup later on.